internal/cmd/connector_test.go

```diff
@@ -72,6 +75,15 @@ func TestConnector_Run(t *testing.T) {
 	ctx := context.Background()
 	runAndWait(ctx, t, srv.Run)
 
+	kubeconfig := path.Join(dir, "kubeconfig")
+	os.Setenv("KUBECONFIG", kubeconfig)
```

Comment:
```suggestion
	t.Setenv("KUBECONFIG", kubeconfig)
```

So that this value is reset when the test ends.

Reply:
Fantastic. Thanks!